dapp/config/default.json

```diff
@@ -30,7 +30,7 @@
         "MultiBridgeMediator": "0x68b762A7a68F6D87Fcf2E2EaF7eF48D00cAa2419"
       },
       "main": {
-        "FuseToken": "0x4162178B78D6985480A308B2190EE5517460406D",
+        "FuseToken": "0x970B9bB2C0444F5E81e9d0eFb84C8ccdcdcAf84d",
```

Comment:
Need also delete the value from `production.json`

Reply:
He doesn’t exist there, so he should get the default

Reply:
oh yeah, right